src/main/scala/d2dadapter/CRC16Lookup8Bit.scala

```diff
@@ -0,0 +1,267 @@
+package edu.berkeley.cs.ucie.digital.d2dadapter
+
+import chisel3._
+
+class CRC16Lookup {
```

Comment:
I tried switching it to an object but I am running into issues with trying to access the data from the CRCGenerator module (it says the wire is not accessible from the CRCGenerator module). I'll try to look more into the documentation to find the right way to do this, but if you have an idea of the issue I am running into I would appreciate some guidance.

Reply:
I've fixed this for you by making the field a method instead, so that the `VecInit` is created at the use site inside the module context (instead of when the singleton object is constructed)